src/Hl7.Fhir.Base/Serialization/FhirJsonBuilderExtensions.cs

```diff
@@ -55,24 +57,24 @@ public static JObject ToJObject(this ITypedElement source, FhirJsonSerialization
             new FhirJsonBuilder(settings).Build(source);
 
         /// <inheritdoc cref="ToJsonAsync(ITypedElement, FhirJsonSerializationSettings)" />
-        [TemporarilyChanged]
+        [TemporarilyChanged] // This works. Remove this attribute after writing new extensions on the pocos
```

Comment:
This looks fine, what else is there to add?

Reply:
Ok, we still use the old JsonBuilder stuff indeed, that needs to go, this is task https://github.com/FirelyTeam/firely-net-sdk/issues/2992